scripts/python/releases-to-appstream.py

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/python3
+import sys
+import requests
+import xml.etree.ElementTree as ET
+import xml.dom.minidom
+from datetime import datetime
+
+OWNER = "Nexus-Mods"
+REPO = "NexusMods.App"
+GITHUB_API = f"https://api.github.com/repos/{OWNER}/{REPO}/releases"
+
+def fetch_releases():
+    releases = []
+    page = 1
+
+    while True:
+        response = requests.get(GITHUB_API, params={"per_page": 100, "page": page})
+        if response.status_code != 200:
+            raise Exception(f"GitHub API error: {response.status_code} - {response.text}")
+
+        page_releases = response.json()
+        if not page_releases:
+            break
+
+        releases.extend(page_releases)
+        page += 1
+
+    return releases
+
+def generate_appstream_xml(releases):
```

Comment:
This function name might confuse someone thinking it's doing substitution into the `metainfo` file at first glance, when it's the `releases` file.

I don't think it's a huge deal, as this is a trivial script, thought I'd just point it out. 
Quite honestly a 1 liner comment at the top of the file would be sufficient.